<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>useWindow</class>
 <widget class="QMainWindow" name="useWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Windows Manager - Main</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>651</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete Instance</string>
     </property>
    </widget>
    <widget class="QComboBox" name="deleteBox">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteBtn">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>âœ“</string>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>651</width>
      <height>281</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>649</width>
       <height>279</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Mode:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="modeBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>330</y>
      <width>401</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="createAction">
   <property name="text">
    <string>Create New Instance</string>
   </property>
  </action>
  <action name="openAction">
   <property name="text">
    <string>Open Existing Instances</string>
   </property>
  </action>
  <action name="quitAction">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="configAction">
   <property name="text">
    <string>Config Menu</string>
   </property>
  </action>
  <action name="locationAction">
   <property name="text">
    <string>Saved .exe Locations</string>
   </property>
  </action>
  <action name="autoplayAction">
   <property name="text">
    <string>Autoplay Speed</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
